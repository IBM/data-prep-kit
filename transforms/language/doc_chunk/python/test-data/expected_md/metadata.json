{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "doc_chunk",
        "job type": "pure python",
        "job id": "job_id",
        "start_time": "2024-07-30 18:27:44",
        "end_time": "2024-07-30 18:27:44",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "chunking_type": "li_markdown",
        "content_column_name": "contents",
        "output_chunk_column_name": "contents",
        "output_jsonpath_column_name": "doc_jsonpath",
        "checkpointing": false,
        "max_files": -1,
        "random_samples": -1,
        "files_to_use": [".parquet"]
    },
    "job_output_stats": {
        "source_files": 1,
        "source_size": 23756,
        "result_files": 1,
        "result_size": 23781,
        "processing_time": 0.014858007431030273,
        "nfiles": 1,
        "nrows": 18,
        "source_doc_count": 1,
        "result_doc_count": 18
    },
    "source": {
        "name": "/Users/dol/scratch/dpk-dev/data-prep-kit-outer/transforms/language/doc_chunk/python/test-data/input_md",
        "type": "path"
    },
    "target": {
        "name": "/Users/dol/scratch/dpk-dev/data-prep-kit-outer/transforms/language/doc_chunk/python/output",
        "type": "path"
    }
}
