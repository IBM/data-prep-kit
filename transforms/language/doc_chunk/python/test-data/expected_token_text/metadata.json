{
  "pipeline": "pipeline_id",
  "job details": {
    "job category": "preprocessing",
    "job name": "doc_chunk",
    "job type": "pure python",
    "job id": "job_id",
    "start_time": "2024-10-04 14:00:40",
    "end_time": "2024-10-04 14:00:41",
    "status": "success"
  },
  "code": {
    "github": "github",
    "commit_hash": "12345",
    "path": "path"
  },
  "job_input_params": {
    "chunking_type": "li_token_text",
    "content_column_name": "contents",
    "doc_id_column_name": "document_id",
    "dl_min_chunk_len": null,
    "output_chunk_column_name": "chunk_text",
    "output_source_doc_id_column_name": "source_document_id",
    "output_jsonpath_column_name": "doc_jsonpath",
    "output_pageno_column_name": "page_number",
    "output_bbox_column_name": "bbox",
    "chunk_size_tokens": 128,
    "chunk_overlap_tokens": 30,
    "checkpointing": false,
    "max_files": -1,
    "random_samples": -1,
    "files_to_use": [
      ".parquet"
    ],
    "num_processors": 0
  },
  "job_output_stats": {
    "source_files": 1,
    "source_size": 17749,
    "result_files": 1,
    "result_size": 8827,
    "processing_time": 0.194,
    "nfiles": 1,
    "nrows": 10,
    "source_doc_count": 2,
    "result_doc_count": 10
  },
  "source": {
    "name": "/Users/jmcappi/git/ai-foundation/data-prep-kit/transforms/language/doc_chunk/python/test-data/input_token_text",
    "type": "path"
  },
  "target": {
    "name": "/Users/jmcappi/git/ai-foundation/data-prep-kit/transforms/language/doc_chunk/python/output",
    "type": "path"
  }
}