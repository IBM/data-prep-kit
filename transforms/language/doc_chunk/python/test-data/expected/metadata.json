{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "doc_chunk",
        "job type": "pure python",
        "job id": "job_id",
        "start_time": "2024-07-31 21:18:32",
        "end_time": "2024-07-31 21:18:32",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "chunking_type": "dl_json",
        "content_column_name": "contents",
        "output_chunk_column_name": "contents",
        "output_jsonpath_column_name": "doc_jsonpath",
        "output_pageno_column_name": "page_number",
        "output_bbox_column_name": "bbox",
        "checkpointing": false,
        "max_files": -1,
        "random_samples": -1,
        "files_to_use": [".parquet"]
    },
    "job_output_stats": {
        "source_files": 1,
        "source_size": 50276,
        "result_files": 1,
        "result_size": 27400,
        "processing_time": 0.051286935806274414,
        "nfiles": 1,
        "nrows": 88,
        "source_doc_count": 1,
        "result_doc_count": 88
    },
    "source": {
        "name": "/Users/dol/scratch/dpk-dev/data-prep-kit/transforms/language/doc_chunk/python/test-data/input",
        "type": "path"
    },
    "target": {
        "name": "/Users/dol/scratch/dpk-dev/data-prep-kit/transforms/language/doc_chunk/python/output",
        "type": "path"
    }
}
